package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify",
-  "version": "2.3.0",
+  "version": "3.0.0",
```

Comment:
Can you please revert this?

Reply:
yes, I wasn't sure to update it ... I'll do asap, thanks

Reply:
done